tests/aio_model/test_selecting.py

```diff
@@ -13,4 +15,22 @@ async def test__select__w_join(manager):
     ).aio_execute())[0]
 
     assert result.id == beta.id
-    assert result.joined_alpha.id == alpha.id
\ No newline at end of file
+    assert result.joined_alpha.id == alpha.id
+
+
+@pytest.mark.skip
```

Comment:
That's nice you didn't just remove the test and add it with the skip mark for the future. I like it.

Reply:
@kalombos Yes, just not to forget :) But now test is updated and should work via `database.aio_execute(query)`